firefox_puppeteer/ui/windows.py

```diff
@@ -99,6 +99,9 @@ def create_window_instance(self, handle, expected_class=None):
 
         if window_type == 'navigator:browser':
             window = BrowserWindow(lambda: self.marionette, handle)
+        elif window_type == 'Browser:page-info':
+            from .pageinfo.window import PageInfoWindow
```

Comment:
Tested and it's the same problem with maybe the cyclic import between the pageinfo window module and windows.py. We would have to refactor those window classes later, so we might be able to get rid of those local imports.